contracts/p0/Broker.sol

```diff
@@ -68,7 +68,7 @@ contract BrokerP0 is ComponentP0, IBroker {
     ) external returns (ITrade) {
         assert(req.sellAmount > 0);
 
-        address caller = _msgSender();
+        address caller = msg.sender;
```

Comment:
`mload` is higher gas than `caller` so should not copy `msg.sender` into a local variable.

Reply:
Not too important since it's p0. (I realized that later)

Reply:
Yeah I made that change everywhere in P1 but purposefully left P0 as is